dg_test.go

```diff
@@ -289,13 +290,14 @@ func TestDirectedGraph_TwoOrDependenciesResolveSecond(t *testing.T) {
 	assert.NoError(t, d.PushStartingNodes())
 	readyNodes := d.PopReadyNodes()
 	assert.Equals(t, len(readyNodes), 2)
-	assert.SliceContains(t, dependencyNode1, readyNodes)
-	assert.SliceContains(t, dependencyNode2, readyNodes)
+	assert.MapContainsKey(t, dependencyNode1.ID(), readyNodes)
+	assert.MapContainsKey(t, dependencyNode2.ID(), readyNodes)
 	// Resolve one node: dependencyNode2
 	assert.NoError(t, dependencyNode2.ResolveNode(dgraph.Resolved))
 	readyNodes = d.PopReadyNodes()
 	assert.Equals(t, len(readyNodes), 1)
-	assert.Equals(t, readyNodes[0].ID(), dependentNode.ID())
+	assert.MapContainsKey(t, dependentNode.ID(), readyNodes)
+
 }
```

Comment:
Do you want this blank line here?  Ditto line 345, et al.

Reply:
Removed. It was a mistake in the find/replace.